src/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ ELSEIF (UNIX)
     ELSEIF (CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
         SET(LIBS_SYSTEM compat pthread)
     ELSE()
```

Comment:
Instead of modifying ELSE add `ELSEIF (CMAKE_SYSTEM_NAME MATCHES "Android") SET(LIBS_SYSTEM c dl)`

Reply:
handled